closure/bin/build/jscompiler.py

```diff
@@ -97,12 +97,16 @@ def _GetJsCompilerArgs(compiler_jar_path, java_version, source_paths,
   # Add the application JAR.
   args += ['-jar', compiler_jar_path]
 
-  for path in source_paths:
-    args += ['--js', path]
+  #collect list of compiler flags
+  compiler_flags.extend(['--js ' + path for path in source_paths])
```

Comment:
`compiler_flags` can be `None` here and throw. You'll need to initialize it in that case.